tasks/create_ordered_kafka_groups.yml

```diff
@@ -0,0 +1,35 @@
+---
+# Run these tasks over kafka_broker group with serial: 1
```

Comment:
delete this file, its used in upgrade

Reply:
actually nvrm i guess you can upgrade 53x to 53y, might need a test case for that

Reply:
nah just delete it, easier